### DISCUSSION TITLE
Strange "Route missed" although on route

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#663] [Q&A] [answered]
Navigated ACT/TET perfectly with Cruiser for almost 2 weeks in Portugal. Only one minor issue:
It happened a few times that “Route missed” appeared, even though I was driving exactly on the route. 

<img width="420" src="https://github.com/user-attachments/assets/402b3eac-4164-41c2-81ef-5a1818631a82"/>
(in this example not really offroad 😉)

I then always just drove on as planned. Only after a few hundred meters did Cruiser automatically synchronize itself again. 

Because I assumed that the GPS signal was perhaps inaccurate and was off the route, I changed the distance from 80m to 300m and the accuracy from 50m to 30m. 

<img width="200" src="https://github.com/user-attachments/assets/f3746a79-78e1-4d0f-b3c5-bac04c0f9f25"/> . . . <img width="200" src="https://github.com/user-attachments/assets/b129e126-4c84-4151-843d-a3a4c6fb1958"/>

After that, the problem no longer occurred - although I'm not sure if this was due to the changes.

**Comment by devemux86** [comment]
@WolfgangZ42 Thanks for the report.

GPS recording filter settings (image 2) affect only the recording of the GPS track, not navigation.

The off route detection in navigation depends on the recalculate route options in the 1st image.

> Because I assumed that the GPS signal was perhaps inaccurate and was off the route

This is correct and also demonstrated [here](https://github.com/devemux86/cruiser/discussions/650), that GPS may often report wrong locations a large distance away.
Although Cruiser waits a bit before warning for off route.

- What location service have you selected, Android or Google?
- You can try Kalman filter in location advanced settings.

In such cases it's nice to record your track with **all filters turned off** in order to capture everything that GPS reports.

**Reply by WolfgangZ42** [reply]
Location Service = Android and Kalman Filter is set. 
Should it happen again I'll try to record it according to your tipps - thanks!

**Comment by devemux86** [comment]
> I changed the distance from 80m to 300m
> After that, the problem no longer occurred

@WolfgangZ42 Note that this distance setting for driving has default value 120m.

**Reply by WolfgangZ42** [reply]
> has default value 120m

I set it to 80m before, now 300m - should I take the default?

**Reply by devemux86** [reply, marked as answer]
@WolfgangZ42 Better return to the default 120m and see how it works.
